xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -20,6 +20,9 @@
 #if defined(TARGET_DARWIN_OSX)
 #include "platform/darwin/osx/smc.h"
 #endif
+#if defined(TARGET_WINDOWS)
```

Comment:
please avoid using ifdefs in common sources

Reply:
cannot be easily avoided

Reply:
but possible. nobody said that it's easy. we did a lot of work to clean up the code from ifdefs, please don't produce them again

Reply:
but possible. nobody said that it's easy. we did a lot of work to clean up the code from ifdefs, please don't produce them again

Reply:
Done